cmd/commands/errors.go

```diff
@@ -1,53 +1,53 @@
 package commands
 
 import (
-	"fmt"
 	"github.com/pkg/errors"
 	"strings"
 )
 
-// Joins and formats results.
+// Joins and formats results, with the provided formatter.
+// The result is skipped if the formatter returns an empty string.
 // Returns nil if all errors are nil.
-// If there is only one non-nil error, the  error is returned as-is.
-func MergeResults(input []error) error {
-	stats := computeStats(input)
+func MergeResults(results []CorrelatedResult, formatter func(CorrelatedResult) string) error {
```

Comment:
It might be nice to pass the input and error directly to the formatter function rather than forcing users to understand what the CorrelatedResult type is.

Reply:
Not sure about that change, though. Would `ApplyInParallel` still return `CorrelatedResult`s in your opinion?

Reply:
Having it exposed on one side and not the other would feel a bit strange to me.